netbox/ipam/tests/test_api.py

```diff
@@ -662,6 +665,107 @@ def test_create_multiple_available_ips(self):
         self.assertEqual(len(response.data), 8)
 
 
+class ParallelAPITests(APITransactionTestCase):
```

Comment:
Let's avoid deviating too much from the existing test structure. I propose:

1. Move this class to `utilities.testing.ParallelAPITestCase` (keep `APITransactionTestCase` as it is)
2. Create `PrefixParallelTest` and `IPAddressParallelTest` in `ipam/tests/test_api.py`
3. Move the two `test_*` methods to their respective test classes

This will ensure the new test cases are immediately distinguishable, and allow for potential reuse of `ParallelAPITestCase` in the future. (Ideally, we'd incorporate these methods within the existing test cases but that doesn't seem feasible without converting the TestCase to a TransactionTestCase.) What do you think?